Dockerfile

```diff
@@ -47,16 +45,10 @@ RUN apt-get update -y && \
   ./aws/install && \
   rm -rf aws && \
   # Add additional apt repos all at once
-  echo "deb https://deb.nodesource.com/node_14.x $(lsb_release -cs) main"                         | tee /etc/apt/sources.list.d/node.list             && \
-  echo "deb https://dl.yarnpkg.com/debian/ stable main"                                           | tee /etc/apt/sources.list.d/yarn.list             && \
   echo "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"      | tee /etc/apt/sources.list.d/docker.list           && \
   # Install second wave of dependencies
   apt-get update -y && \
-  apt-get install -y \
-  docker-ce \
-  amazon-ecr-credential-helper \
```

Comment:
does the production pro backend still use this?

Reply:
the change to remove it is merged into main, but it's not running.
prod is waiting for cli to be released. So if this merges, it will be fine.